server/submission/definitions.js

```diff
@@ -29,14 +29,10 @@ const typeDefs = `
         
     extend type Manuscript {
       # todo: these should be handled through teams
-      suggestedSeniorEditors: [EditorUser]
-      opposedSeniorEditors: [EditorUser]
-      suggestedReviewingEditors: [EditorUser]
-      opposedReviewingEditors: [EditorUser]
       suggestedReviewers: [SuggestedReviewer]
       opposedReviewers: [OpposedReviewer]
       author: Alias
-    
+
       # todo: these should be handled through notes
       opposedSeniorEditorsReason: String
```

Comment:
Something tells me that the "Reason" should be stored on the link too. Just flagging - not suggesting it gets done as part of this PR.